demo/nexmark/nexmark.sql

```diff
@@ -0,0 +1,60 @@
+CREATE TABLE person (
+    id BIGINT,
+    name VARCHAR,
+    emailAddress VARCHAR,
+    creditCard VARCHAR,
+    city VARCHAR,
+    state VARCHAR,
+    date_time TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
+    extra  VARCHAR
+);
+CREATE TABLE auction (
+    id  BIGINT,
+    itemName  VARCHAR,
+    description  VARCHAR,
+    initialBid  BIGINT,
+    reserve  BIGINT,
+    date_time  TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
+    expires  TIMESTAMP(3),
+    seller  BIGINT,
+    category  BIGINT,
+    extra  VARCHAR
+);
+CREATE TABLE bid (
+    auction  BIGINT,
+    bidder  BIGINT,
+    price  BIGINT,
+    channel  VARCHAR,
+    url  VARCHAR,
+    date_time TIMESTAMP(3), -- NOT NULL LATENESS INTERVAL 4 SECONDS,
+    extra  VARCHAR
+);
+-- -------------------------------------------------------------------------------------------------
+-- Query 9: Winning Bids (Not in original suite)
+-- -------------------------------------------------------------------------------------------------
+-- Find the winning bid for each auction.
+-- -------------------------------------------------------------------------------------------------
+
+CREATE VIEW q9 AS
```

Comment:
We'd be executing both queries at once in this benchmark. Maybe leave a note saying they should be split later?

Reply:
fwiw being faster than X and running all queries at once might also be a powerful statement.. if we can

Reply:
The new version I just pushed runs one query at a time, or you can tell it --merge and it will run the requested queries together.

I ran a few queries together and they took about the same time as running just one of them individually.